configs/nri-prometheus-config.yml.sample

```diff
@@ -0,0 +1,27 @@
+integrations:
+  - name: nri-prometheus
+    inventory_source: integration/com.newrelic.prometheus
+    config:
+      # The name of your cluster. It's important to match other New Relic products to relate the data.
+      cluster_name: "my_exporter"
+      
+      #targets:
+      #  - description: Secure etcd example
+      #    urls: ["https://192.168.3.1:2379", "https://192.168.3.2:2379", "https://192.168.3.3:2379"]
+      #    tls_config:
+      #      ca_file_path: "/etc/etcd/etcd-client-ca.crt"
+      #      cert_file_path: "/etc/etcd/etcd-client.crt"
+      #      key_file_path: "/etc/etcd/etcd-client.key"
+
+      # Wether the integration should run in verbose mode or not. Defaults to false.
+      verbose: false
+
+      # The HTTP client timeout when fetching data from endpoints. Defaults to 30s.
+      # scrape_timeout: "30s"
```

Comment:
Maybe it's a good idea to remove this. It doesn't make sense if the integration is being executed by the agent and die as soon as it finishes its work.

Reply:
Why? I removed the  `scrape_duration: "30s"` since the integration dies, but the timeout isn't still useful in case the scraper does not respond?

Reply:
Ah, I confused this with the interval, sorry @paologallinaharbur. 😓